man/gemfile.5.ronn

```diff
@@ -287,7 +303,17 @@ Git repositories support a number of additional options.
   * `branch`, `tag`, and `ref`:
     You `MUST` only specify at most one of these options. The default
     is `:branch => "master"`
+  * For example:
+
+            `git 'https://github.com/rails/rails.git', :branch => '5-0-stable' do`
```

Comment:
Sorry i wasn't terribly clear. I meant 8 spaces from the start of the line or 4 spaces relatively to the start of the previous line.